recipes/sherpas/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "sherpas" %}
+{% set version = "1.0.0" %}
+{% set sha = "215729efbfe969b51626716e470e4570ec1c32a383e1646401f5ce708e8446fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/phylo42/{{ name|lower }}.git
```

Comment:
Is there a reason why `git_url` is used here? We usually recommend using the release, e.g. `https://github.com/phylo42/sherpas/archive/1.0.0.tar.gz`.

Reply:
Dear Natasha,
The main problem is that "releases" archives built via github do not include the GIT submodules configured in the repository. And from what I read n forums, github is not going to change this in the near future.

Until github allows to built releases including submodules, the `git_url` seems a much better option for maintenance, of recipes involving many submodules.

As a workaround I will use update the recipe with the manually built archive: `https://github.com/phylo42/sherpas/releases/download/1.0.0/1.0.0.tar.gz`

The github url is then different (release/download/, not /archive/), is this ok with future auto_bumps ?